src/test/java/seedu/momentum/logic/commands/CommandTestUtil.java

```diff
@@ -75,7 +75,7 @@
      * - the {@code actualModel} matches {@code expectedModel}
      */
     public static void assertCommandSuccess(Command command, Model actualModel, CommandResult expectedCommandResult,
-            Model expectedModel) {
+                                            Model expectedModel) {
```

Comment:
Should the indentation here and in the method below be 4 spaces from the parent comment?

Reply:
I think 8 spaces not 4. Changed to 8 spaces.